awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx

```diff
@@ -143,7 +144,7 @@ function JobTemplateForm({
       playbookHelpers.setValue(0);
       scmHelpers.setValue('');
     },
-    [setProject, projectHelpers, playbookHelpers, scmHelpers]
+    [] // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
This would mean this callback is only called when the project lookup changes, which makes sense. I don't think there's another reason to update. :+1: